grafana-sync.py

```diff
@@ -333,6 +328,78 @@ def auth_keys(base_url, session):
     request.raise_for_status()
     print(request.json())
 
+class GrafanaLogin:
+    def __init__(self, token:str, user:str, password: str, base_url: str):
+        self.session = requests.Session()
+        if token:
+            self.session.headers = {"Authorization": f"Bearer {token}"}
+        else:
+            create_token(base_url, self.session, user=user, password=password)
+class GrafanaClient:
+    def __init__(self, base_url: str, session: GrafanaLogin, overwrite: bool) -> None:
+        super().__init__()
+        self.url = base_url
+        self.session = session.session
+        self.folders = self.__load_folders()
+        self.alerts = self.__load_alerts()
+        self.overwrite = overwrite
+
+    def copy_alerts_from(self, source: 'GrafanaClient'):
+        if self.url.startswith("http"):
+            source_alerts_with_target_folders = source.alerts
+            update_folder = lambda s_folder, t_folder: self.__update_alerts_folders(s_folder, t_folder, source_alerts_with_target_folders)
+            self.__sync_folders_with(source, update_folder)
+            self.__save_alert_to_grafana(source_alerts_with_target_folders)
+        else:
+            self.__save_alerts_to_file(source.folders, source.alerts)
+    def __sync_folders_with(self, source: 'GrafanaClient', update_childs: Callable[[dict, dict], None])-> None:
```

Comment:
please leave space between methods
```suggestion
            self.__save_alerts_to_file(source.folders, source.alerts)

    def __sync_folders_with(self, source: 'GrafanaClient', update_childs: Callable[[dict, dict], None])-> None:
```